react-app/src/components/TopNavigationBar/TopNavigation.css

```diff
@@ -29,6 +29,7 @@ p, ul, div, li, a, img h1, h2, h3, h4, section {
     width: 100%;
     box-sizing: border-box;
     position: fixed;
+    z-index: 10;
```

Comment:
as i mentioned, i recommend to comment this out for now since it might mess with current styling in the dev branch.. we can add it later if necessary